File: admin-frontend/admin-system/src/types/article.ts
export interface Article {
    article_id: number
    title: string
    content: string
    author_id: number
    author_name: string
    publication_time: string
    is_banned: boolean
    img?: string  // 图片 URL 路径，不再是 base64
    address?: string
    like_count?: number
    star_count?: number
    is_reported?: boolean
    report_id?: number
    report_reason?: string
    // 兼容旧字段
    img_url?: string  // 已废弃，使用 img
}

File: bookshop-backend/red-book-back/src/main/java/com/example/back/pojo/Article.java
package com.example.back.pojo;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Article {
    private String txt_type;
    private Integer author_id;
    private String content;
    private String title;

    private Integer article_id;
    private String publication_time;
    private String address;
    private Integer star_count;
    private Integer like_count;

    // 新增 img 字段，用于存储图片的URL路径
    private String img;

    // 新增字段
    private Integer is_review;
    private Integer is_banned;
}